### DISCUSSION TITLE
Add FE indicator about recency of our activity data

### DISCUSSION THREAD

**Discussion by bnelson-czi** [chanzuckerberg/napari-hub#955] [Ideas]
We underwent our first major analytics data outage, which resulted in stale data for a few days. It didn't show on the activity dashboard since it was mid-March and we don't show data on incomplete months. Having said that, it might be useful for end users and devs to see how up-to-date our activity data are, through a tooltip or icon.

We do something similar on our internal Superset dashboards.
![Screenshot 2023-03-20 at 2 36 30 PM](https://user-images.githubusercontent.com/68256044/226471405-6d957322-8af3-4203-8df8-dff6bbfc1dea.png)

**Comment by kandarpksk** [comment]
It seems the age of data would be the same across plugins, so we could consider a common location (such as the footer) to show the information. Another factor is that metrics may appear outside of the activity tab as well (and those may also be out of date).
